jest.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
I guess this file could be written in TypeScript as well.

Reply:
Maybe "@types/jest" needs to be installed as a devDependency.

Reply:
Changed.